student.rb

```diff
@@ -4,6 +4,7 @@ class Student < Person
   def initialize(classroom, age, name = 'Unknown', parents_permission = 'true')
     super(age, name, parents_permission)
     @classroom = classroom
+    @classroom.add_student(self)
   end
 
```

Comment:
- [ ] Kindly make sure to create a method that represents the `belong-to` relationship between the Student and Classroom classes (a student belongs to a classroom)

```ruby
def classroom=(classroom)
..
..
```